redcap_hooks.php

```diff
@@ -108,6 +114,13 @@ function redcap_every_page_before_render($project_id = null)
 }
 
 
+// redcap_pdf (REDCap >= 9.5.0)
```

Comment:
Spaces around the parameters for the functions below is inconsistent with the rest of the file.

Reply:
Ah yes, good point. I copied this straight from the REDCap Hooks page so the formatting is a little different. I'll revise it.